dockerfiles/registrar.Dockerfile

```
@@ -0,0 +1,95 @@
+FROM ubuntu:focal as app
+
+# ENV variables for Python 3.12 support
+ARG PYTHON_VERSION=3.12
+ENV TZ=UTC
+ENV TERM=xterm-256color
+ENV DEBIAN_FRONTEND=noninteractive
+
+# software-properties-common is needed to setup Python 3.12 env
+RUN apt-get update && \
+  apt-get install -y software-properties-common && \
+  apt-add-repository -y ppa:deadsnakes/ppa
+
+# System requirements.
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -qy \
```

we've already defined the flag above so it is not needed here.
```suggestion
RUN apt-get install -qy \
```